modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/seedlist/types.scala

```diff
@@ -1,20 +1,40 @@
 package org.tessellation.sdk.infrastructure.seedlist
 
 import org.tessellation.schema.peer.PeerId
-import org.tessellation.security.hex.Hex
+import org.tessellation.schema.trust._
 
-import fs2.data.csv.RowDecoder
+import eu.timepit.refined.auto._
+import fs2.data.csv._
 import fs2.data.csv.generic.semiauto.deriveRowDecoder
 
 object types {
 
-  case class SeedlistCSVEntry(id: String) {
-
-    def toPeerId: PeerId = PeerId(Hex(id))
-  }
+  case class SeedlistCSVEntry(
+    id: PeerId,
+    ipAddress: String,
```

Comment:
These types here are not backwards compatible, if an entry is missing an IpAddress (i.e. the current entries) you are enforcing an expected value here.

This entire PR appears not to be backwards compatible with existing seedlist. It should load the existing file throughout here independently, so we don't need to perform an update at the same time the code is changed.

These should be changed to optional fields, and the rest of the code should reflect that.

Reply:
NVM re: backwards compat, after more discussion offline, this appears to be correct, but the types still need to change to optional for clarity and test added for showing mixed line parsing.

Reply:
`SeedlistCSVEntry` has been changed to:

```
  case class SeedlistCSVEntry(
    id: PeerId,
    ipAddress: Option[String],
    alias: Option[String],
    bias: Option[TrustValueRefined]
  )
```

The tests and decoder have been updated appropriately. I also added a test for mixed formats in the same seedlist file.